### DISCUSSION TITLE
Resource localization new project template

### DISCUSSION THREAD

**Discussion by maaxe** [unoplatform/uno#11208] [Q&A] [answered]
I am trying to migrate my old UWP based Uno project to WinUI and the new project template but running into some problems with localization. I was previously using `Windows.ApplicationModel.Resources.ResourceLoader.GetForCurrentView()`  in a ValueConverter. 

I changed to `GetForViewIndependentUse("MyLibraryName\\Resources") ` to get the ResourceLoader from a worker thread and this works fine for Windows head. But for the WPF head `resourceLoader.GetString("SomeLocalizedResource")` does not find anything and returns an empty string. 

What am I missing here?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report, this is being worked on in https://github.com/unoplatform/uno/pull/11149, an update will be available shortly.
